core-v1/src/main/scala/dsl/simple.scala

```diff
@@ -2,12 +2,16 @@ package com.rallyhealth.vapors.v1
 
 package dsl
 
-case object simple extends FullDsl with UnwrappedBuildExprDsl with SimpleRunDsl with ShowOPDsl {
+case object simple extends FullDsl with UnwrappedBuildExprDsl with CachingRunDsl with ShowOPDsl {
 
-  final case object withDebugInfo extends UnwrappedBuildExprDsl with SimpleRunDsl with DebugSourceInfoDsl
+  final case object withDebugInfo extends UnwrappedBuildExprDsl with CachingRunDsl with DebugSourceInfoDsl
 
-  final case object justified extends FullDsl with JustifiedBuildExprDsl with SimpleRunDsl with ShowOPDsl {
+  final case object justified extends FullDsl with JustifiedBuildExprDsl with CachingRunDsl with ShowOPDsl {
 
-    final case object withDebugInfo extends FullDsl with JustifiedBuildExprDsl with SimpleRunDsl with DebugSourceInfoDsl
+    final case object withDebugInfo
+      extends FullDsl
+      with JustifiedBuildExprDsl
+      with CachingRunDsl
+      with DebugSourceInfoDsl
   }
 }
```

Comment:
This change uses caching by default (`dsl.simple` now uses caching). I could alternatively create a `dsl.caching` import to allow the user to decide. I think this might be preferable... maybe `dsl.uncached` / `dsl.caching.mutable` / `dsl.caching.parallel` could be the future state for the DSLs?